crates/primitives/src/bits.rs

```diff
@@ -1,4 +1,6 @@
 //! Fixed hash types
+#![allow(clippy::incorrect_clone_impl_on_copy_type)]
```

Comment:
where's this from?

Reply:
It is on bits.

Reply:
Should dissapear when we introduce alloy types